City-of-Helsinki/drupal-infofinland
# IFU-313: Language configurations

Drupal installation now works in 12 languages.

To test:

- Run `make drush-cim`
- Check that all languages work and language list matches those in the old site
